staging/src/k8s.io/apiextensions-apiserver/test/integration/ratcheting_test.go

```diff
@@ -256,15 +250,12 @@ func (u updateMyCRDV1Beta1Schema) Do(ctx *ratchetingTestContext) error {
 		// Keep trying to create an invalid instance of the CRD until we
 		// get an error containing the ResourceVersion we are looking for
 		//
-		counter := 0
 		return wait.PollUntilContextCancel(context.TODO(), 100*time.Millisecond, true, func(_ context.Context) (done bool, err error) {
-			counter += 1
 			err = applyPatchOperation{
 				gvr:  myCRDV1Beta1,
 				name: "sentinel-resource",
 				patch: map[string]interface{}{
-					// Just keep using different values
-					sentinelName: fmt.Sprintf("invalid %v %v", uuidString, counter),
+					sentinelName: "doesntmatteralwaysfails",
```

Comment:
will passing the same value ever result in a no-op patch that will short-circuit validation we were trying to trigger?

Reply:
I'd suggest keeping this a changing value, but not making it the uuid (incrementing value, math.Rand, current time nanos, etc)

Reply:
> will passing the same value ever result in a no-op patch that will short-circuit validation we were trying to trigger?

Since the injected field when used always rejected its value it is impossible for there to be an existing object with the same value for this field.

> I'd suggest keeping this a changing value, but not making it the uuid (incrementing value, math.Rand, current time nanos, etc)

Changed to use the counter for added certainty